frontend/src/utils/api.js

```diff
@@ -250,6 +252,55 @@ function deleteReferral(candidateID: string) {
   }).then(res => res.json())
 }
 
+function registerUser(email: string, password: string, role: string) {
+  console.log(`Creating new user: ${email}`)
+  return fetch(`${AUTH_API_URL}/register`, {
+    method: 'POST',
+    body: JSON.stringify({
+      email,
+      password,
+      role
+    }),
+    headers: {
+      'content-type': 'application/json'
+    },
+    mode: 'cors'
+  }).then(res => res.json())
+}
+
+function loginUser(email: string, password: string) {
+  console.log(`Logging in user ${email}`)
+  return fetch(`${AUTH_API_URL}/login`, {
+    method: 'POST',
+    body: JSON.stringify({
+      email,
+      password
+    }),
+    headers: {
+      'content-type': 'application/json'
+    },
+    mode: 'cors'
+  }).then(res => res.json())
+}
+
+function loginGoogleUser(tokenId: string) {
+  console.log(`Logging in user ${tokenId} with Google Auth`)
+  try {
+    return fetch(`${AUTH_API_URL}/google`, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json'
+      },
+      body: JSON.stringify({
+        tokenId: tokenId,
+        role: 'member'
+      })
+    })
+  } catch (err) {
+    console.log(err)
```

Comment:
just as a general comment, is there a reason why we log the error here and not in `loginUser`, etc? Also this seems like we silently fail on the frontend — that's definitely not good UX for a user